easybuild/easyconfigs/t/text-unidecode/text-unidecode-1.3-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonBundle'
```

Comment:
This needs to be a python package

Reply:
I believe @boegel told me to prioritize 'PythonBundle' over 'PythonPackage', but I can update this if you think 'PythonPackage' is better here.

Reply:
Yes for single Python packages it is better to use PythonPackage because otherwise the module will be a bit "weird" as the python package will turn up as a "extension" in the module file that "clutters" `module spider` output for LMod.

I guess the reason for preferring PythonBundle is rather to avoid having too many PythonPackage ECs that are not shared between ECs, so it makes sense to put multiple packages into a bundle, like with Spyder. But for single packages I can't see an advantage for the Bundle.